proofs/cbmc/adrs_set_hash_address/Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) The mlkem-native project authors
+# SPDX-License-Identifier: Apache-2.0 OR ISC OR MIT
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = adrs_set_hash_address_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = adrs_set_hash_address
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += adrs_set_hash_address_harness.c
```

Comment:
This should be a project source. Since your function is in a header, you need to use another c file that includes that header.